experiments/samples/world-cultures-geography/fileset/ebraille/WCAG-ch1-2.html

```diff
@@ -0,0 +1,787 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" lang="en-Brai-US" xml:lang="en-Brai-US">
+    <head>
+    	<meta charset="UTF-8"/>
+    	<title>Chapter 1 - Part 2</title>
+    </head>
+    <body epub:type="bodymatter">
+        <section epub:type="part" class="unit" id="d18656e63">
+            <span epub:type="pagebreak" role="doc-pagebreak" title="2" id="p2"/>
+            <header><h1 id="d20258e55-0" class="part"><span class="label" epub:type="label">⠠⠠⠥⠝⠊⠞</span>
+                    <span class="ordinal" epub:type="ordinal">⠼⠁</span>⠤⠤⠠⠠⠔⠞⠗⠕⠙⠥⠉⠰⠝ ⠠⠠⠖⠠⠠⠸⠺ ⠠⠠⠉⠥⠇⠞⠥⠗⠑⠎ ⠠⠠⠯                     ⠠⠠⠛⠑⠕⠛⠗⠁⠏⠓⠽</h1></header>
+            <figure>
+                <img src="images/wcag-p2-1.jpg" alt="Satellite view of the Earth at night. This image spans pages 2 and 3" id="wcag-p2-1" aria-describedby="pnote-p2-1"/>
```

Comment:
Did we say we want to translated the alt attributes?

By the way, how did you produce the braille? If you used DAISY Pipeline, I could do a quick fix to also handle the necessary attributes.

Reply:
I don't remember how the braille was produced anymore. Too long ago and I didn't create the sample. I just grabbed the braille rendition of the epub here: https://github.com/IDPF/epub3-samples/tree/main/30/WCAG

I didn't make any changes to the content of the publication, so I'm sure there are all kinds of other issues. I figure we can clean that up later, as it'll need someone familiar with UEB to do it right.

Reply:
Yes that seems like something that came from Pipeline.

Is it OK that if I discover issues like `alt` attribute one, I make a note here so we don't forget to fix it later?